product-updates/knowledge-center/view-submission-history.html

```diff
@@ -308,30 +308,157 @@ <h3 class="usa-process-list__heading">Search for the reporting period to view</h
+      <h3 class="usa-process-list__heading">Select the "Submission History" tab</h3>
   <br>
       <p>After searching for the reporting period, a new button group labelled 'Current Submission' and 'Submission History’ will appear below the 'Search' button.</p>
       <p>The button ‘Current Submission’ is default selected. Navigate to the ‘Submission History’ button to view the data files previously submitted through TDP.</p>
       <a href="../img/getting-started/submission-history.png" data-lity data-lity-desc="Submission history page listing five visible previous submissions with more accessible via a pagination control below the current page of the table">
       <img src="../img/getting-started/submission-history.png" alt="Submission history page listing five visible previous submissions with more accessible via a pagination control below the current page of the table">
         </a>
-      The 'Submission History' table includes the following fields:
-      <ul>
-        <li>Submitted On: The date and time the data file(s) were submitted</li>
-        <li>Submitted By: The user who submitted the data file(s)</li>
-        <li>File Name: The name of the data file(s) are clickable and downloads the file</li>
-      </ul>
+ 
     </li>
 
-    <li class="usa-process-list__item">
-      <h3 class="usa-process-list__heading">How to retrieve your submitted data</h3>
-      <br>
-      <p>To access a file previously uploaded to TDP from 'Submission History', you can click the link in the ‘File Name’ column. Note: TDP will auto-rename files to specify the fiscal period and section (e.g. 2021.Q1.Active Case Data), so depending on your original file-naming conventions, the files you download may have a different name.</p>
-      <a href="../img/getting-started/click-file-name-download.png" data-lity data-lity-desc="Submission history page emphasizing the ability to click on the file name of a previously submitted file to download it">
-      <img src="../img/getting-started/click-file-name-download.png" alt="Submission history page emphasizing the ability to click on the file name of a previously submitted file to download it">
-      </a>
+
   </ol>
 
+   
+<hr style="border: none;
+    height: 1px;
+    background: #000;
+    margin-top: 32px;
+    background: repeating-linear-gradient(90deg,#000,#000 6px,transparent 6px,transparent 12px);">
+    <div style="min-height: 22px;"></div> <div style="min-height: 22px;"></div> 
+
+   <h2>Download Error Reports and Previously Submitted Data Files</h2>
+  <div style="min-height: 22px;"></div>
+    
+
+      <p>To access error reports for each file you submit. Click on the link in the ‘Error Report’ column to view errors and data feedback associated with that version of the file. Depending on the completeness of your data and how soon you access the Submission History tab after submitting a file you may see other values in this column including "Pending" and "No Errors".</p>
```

Comment:
we really should be clear about what "no errors" means. there may be additional data quality errors based on OFA's further review of the data. @reitermb